pkg/helm/watcher/controller/helm_watcher.go

```diff
@@ -75,16 +89,22 @@ func (r *HelmWatcherReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 	values := make(cache.ValueMap)
 
 	for _, chart := range charts {
+		if v, err := r.checkForNewVersion(ctx, chart); err != nil {
+			log.Error(err, "checking for new versions failed")
+		} else if v != "" {
+			log.Info("sending notification event for new version", "version", v)
+			r.event(ctx, &repository, repository.Status.Artifact.Revision, "info", fmt.Sprintf("New version available for profile %s with version %s", chart.Name, v))
+		}
+
```

Comment:
do we want to alert for every profile? Or just when a profile that is installed has a new version? cc @bigkevmcd 

I don't know how useful it is to know a new versions of profile X is available, if I'm not evening using X :thinking: or perhaps adjust it to only alert `majors`(and maybe `minors`?)? Just thinking about how noisy this is

Reply:
So, yeah I had the same question. I'll create some tickets about this when I get there. :D But in general, yes for now. because it's not quite easy to get what's installed or not.

Reply:
This is a really good question, and actually, a tricker one to answer than it seems...

I'd say "only installed ones", but determining that is harder than it looks, let's get it going for all, and then work out how to filter for "only installed ones".

Reply:
> So, yeah I had the same question. I'll create some tickets about this when I get there. :D But in general, yes for now. Because it's not quite easy to get what's installed or not.

yeah leaving it as is and adding tickets sounds good. @bigkevmcd pointed out on slack:
```
But, what does it mean to filter for installed?
4:06
Installed in this cluster ?
4:06
Installed in a repo ?
4:06
Installed in any of a set of clusters ?
```

lets capture this and we can decide how to proceed

Reply:
cc @Himangini